lib/index.js

```diff
@@ -13,6 +13,47 @@ var parseConfig 		= require('./parseConfig');
 var Falafel = function () {
 
 	return {
+
+		//This should be run independantly (i.e. no other falafel instance)
+		generateJsonSchema: function (options) {
+
+			options.isLegacy = isLegacyConnector(options.directory);
```

Comment:
this checks feels a bit unneeded, as connectors builder already knows at this point if it's processing a manual connector or not.

Reply:
Right, but, that's assuming that "connectors builder already knows" is the only context?

If the method were to be used outside of the connectors builder context, then I think having such a check would be a good guard.

Relatedly, do you think I should wrap the code in a try/catch, and return something else if there's an error or log it?

Reply:
yeah in other contexts it could be useful, so maybe could be an option inside `options`. By the way, in connector builder context, it would be useful if this method handled built-in connectors better:
- if there is no schema.js, assert that there must be connectors.json file, and parse it and return it
if there is schema.js, use that similarly as for manual connectors.

but that could be a separate task.

as for try/catch - well, no need. Though returning a promise could be nice, to signify that this is a relatively long I/O operation.

Reply:
to add some more context: ideally builtin connectors would have only schema.js (+ other files referenced by it, if needed), after we migrate them away from connectors.json

Reply:
I think I have addressed this. See `falafel.js` file under tests.

Option to bypass manual check has been added in the form of `bypassManualCheck` flag.